utils/utils.ts

```diff
@@ -192,7 +192,7 @@ export const chainIdExportsAddresses = (chainId: string) => {
 
 // CONTRACT ADDRESSES
 export const writeContractAddresses = (chainId: string, addresses: Array<[string, string]>) => {
-    const contractsJSON = fs.readFileSync('./data/contracts.json')
+    const contractsJSON = fs.readFileSync('utils/data/contracts.json')
```

Comment:
I don't think (and the same change belowe) this is correct, it should be pointing at the data folder